course/src/isometry.cc

```diff
@@ -0,0 +1,115 @@
+#include <isometry.h>
+
+namespace ekumen {
+  namespace math {
+
+  const Vector3 Vector3::kUnitX(1.,0,0);
+  const Vector3 Vector3::kUnitY(0,1.,0);
+  const Vector3 Vector3::kUnitZ(0,0,1.);
+  const Vector3 Vector3::kZero(0,0,0);
+
+  Vector3 Vector3::operator + (const Vector3 & obj) const
+  {
+    return Vector3( x_ + obj.x(), y_ + obj.y(),z_ +  obj.z());
```

Comment:
Implement this and other operators using their  `+=` cousin and friends. You usually need both, and that way you only implement the operation once.